Solutions/Marain.Services.Tenancy/Marain/Services/Tenancy/IMarainServiceTenancyHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿// <copyright file="IMarainServiceTenancyHelper.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.Services.Tenancy
+{
+    using System.Threading.Tasks;
+    using Corvus.Tenancy;
+    using Corvus.Tenancy.Exceptions;
+    using Menes.Exceptions;
+
+    /// <summary>
+    /// Provides methods required by Marain services to validate and work with tenants.
+    /// </summary>
+    public interface IMarainServiceTenancyHelper
```

Comment:
I have to say I don't like `Helper`

Isn't this just `IMarainServiceTenancyManagement` (to reflect `ITenantManagement`)